opennmt/models/sequence_to_sequence.py

```diff
@@ -483,7 +484,7 @@ def replace_unknown_target(target_tokens,
       x=aligned_source_tokens,
       y=target_tokens)
 
-def _add_noise(tokens, lengths, params, subword_token):
+def _add_noise(tokens, lengths, params, subword_token, is_spacer=None):
```

Comment:
Could you update the code that calls `_add_noise` in this file (line 271 in the latest revision)? It should read the new parameter and pass it to this function.